tern/analyze/docker/dockerfile.py

```diff
@@ -333,3 +334,47 @@ def write_locked_dockerfile(dfile, destination=None):
         file_name = constants.locked_dockerfile
     with open(file_name, 'w') as f:
         f.write(dfile)
+
+
+def check_multistage_dockerfile(dfobj):
+    """Given a dockerfile object, return if it is a multistage dockerfile,
+    and the index of FROM line in the dfobj structure."""
```

Comment:
small nit: `and the indexes of the FROM lines` since there will be multiple from lines in a multistage dockerfile.